### PR TITLE
fix missing status code in SignalR

### PR DESCRIPTION
**Pull request by juniwang**
Fix minor validation errors in swagger: 
- Missing http code in CreateOrUpdate API.
- Missing default http status code

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/signalr/resource-manager/Microsoft.SignalRService/stable/2018-10-01/signalr.json [hunk 1]
```diff
@@ -619,6 +704,20 @@
         }
       }
     },
+    "LogSpecification": {
```
Comment (MyronFanQiu) [inline specification/signalr/resource-manager/Microsoft.SignalRService/stable/2018-10-01/signalr.json]: For ARM's review.

File: specification/signalr/resource-manager/Microsoft.SignalRService/stable/2018-10-01/signalr.json [hunk 2]
```diff
@@ -641,6 +740,45 @@
...
+        },
+        "message": {
+          "description": "A message that describes the error in detail and provides debugging information.",
+          "type": "string"
+        },
+        "target": {
+          "description": "The target of the particular error (for example, the name of the property in error).",
+          "type": "string"
+        },
+        "details": {
```
Comment (MyronFanQiu) [inline specification/signalr/resource-manager/Microsoft.SignalRService/stable/2018-10-01/signalr.json]: not common. But SDKs are generated correctly. It's fine to me.

**Review by mmyyrroonn — APPROVED** [review]
LGTM. But it add property into stable version. Need ARM's signoff

**Comment by MyronFanQiu** [comment]
@pilor Hello. Can you take a look at this PR?

**Comment by pilor** [comment]
> @pilor Hello. Can you take a look at this PR?

Read-only property in the operations API is fine

**Review by pilor — APPROVED** [review]

**Pull request merged**